app/src/app/app.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import {products} from "./shared/mock/products";
+import {Products} from './shared/models/products.interface'
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'app';
+  public products:Products[] = products;
```

Comment:
It's not supposed to store anything, move it to the appropriate component ("products", if I'm not mistaken).